apps/rellotge/README.md

```diff
@@ -0,0 +1,5 @@
+# Rellotge, una aplicació per llegir l'hora en català
```

Comment:
I'm happy with this but wanted to check if you had any plans for multilingual apps @gfwilliams

Reply:
Yes, that's fine, although in an ideal world, an english translation for the readme would be nice as well so English readers know what's going on?

For this clock the whole point is it's Catalan, so in the app it makes sense. However in general for multilingual apps there's a whole system we set up (with the `/*LANG*/` comments) to allow apps to be automatically translated - so I'd prefer more general purpose apps to have translations done that way.

Reply:
Sounds like a plan - what do you think @enricpineda?

Reply:
I have no problem with that! I'll put an english translation of the information in the README file, as well as an Esperanto translation.

Reply:
Thanks! I'll look at merging soon once you're happy with the below comments - thanks for looking over too @jordimas